src/Service/MailTemplate/ComplexMacroHelper.php

```diff
@@ -41,6 +42,7 @@ public function buildMacros($entity): array
         $values += match (true) {
             $entity instanceof CaseEntity => $this->buildCaseMacros($entity),
             $entity instanceof Agenda => $this->buildAgendaMacros($entity),
+            $entity instanceof HearingPost => $this->buildCaseMacros($entity->getHearing()->getCaseEntity()),
```

Comment:
For consistency, it may be nicer to add a `buildHearingPostMacros` function and let it call `buildCaseMacros` and whatever must be done (in the future).

`$entity instanceof AgendaBroadcast` breaks this consistency, but we can easily fix that.

Reply:
Fixed in fc04073db7cf0c2a97346a8bf0ca9fbe13a6b34e.